sourcemash/categorize.py

```diff
@@ -1,7 +1,9 @@
 from collections import Counter
 from string import punctuation
 
-def categorizeItem(title, text, category_dict):
+def categorize_item(title, text, category_dict):
```

Comment:
No. The categorization algorithm is its own unit that exists distinctly from the scraper, and it's precondition is string article title, string article text, and dictionary/list of possible categories.